hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/rmnode/RMNodeImpl.java

```diff
@@ -1325,6 +1325,10 @@ public void transition(RMNodeImpl rmNode, RMNodeEvent event) {
           .handle(
               new NodeResourceUpdateSchedulerEvent(rmNode, ResourceOption
                   .newInstance(rmNode.totalCapability, 0)));
+
```

Comment:
Cosmetic:
Can you add a comment maybe suggesting that you're marking recommissioning nodes as usable or maybe even why you're marking them as usable. Just a thought.

Reply:
Done